internal/agent/hooks/runstage.go

```diff
@@ -16,7 +16,7 @@ func (r RunStage) Run(c config.Config, _ v1.Spec) error {
 	if err != nil {
 		cfg.Logger.Errorf("Error reading config: %s\n", err)
 	}
-	_ = utils.RunStage(&cfg.Config, "kairos-install.after", cfg.Strict, cfg.CloudInitPaths...)
+	_ = utils.RunStage(cfg, "kairos-install.after")
```

Comment:
as we now pass the full cfg in here, we no longer need to add the strict or cloudinitpaths as a separate entity, we can just use the cfg object ot get those :D